apps/github-cascading-app/src/cascading/cascading.ts

```diff
@@ -143,10 +144,8 @@ export abstract class Cascading {
       canBeMerged: pullRequest?.mergeable ?? true,
       id: pullRequest?.id || '',
       originBranchName: pullRequest?.originBranchName || '',
-      // eslint-disable-next-line @typescript-eslint/naming-convention
-      CANCEL_RETRIGGER_CASCADING_MARK,
-      // eslint-disable-next-line @typescript-eslint/naming-convention
-      CANCEL_BYPASS_REVIEWERS_MARK
+      cancelRetriggerCascadingMark,
+      cancelBypassReviewersMark
```

Comment:
```suggestion
      cancelRetriggerCascadingMark: CANCEL_RETRIGGER_CASCADING_MARK,
      cancelBypassReviewersMark: CANCEL_BYPASS_REVIEWERS_MARK
```
I think it makes sense to have the full uppercase for the constants and the camelCase for the inputs